src/GenericForm.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import './App.css';
+import Input from './Input.js';
+import Textarea from './Textarea.js';
+import Title from './Title.js';
+import Radio from './Radio.js';
+import Select from './Select.js';
+import Password from './Password.js';
+
+class GenericForm extends Component {
+
+    render() {
+
+        let childNodes = [];
+
+        let fields = this.props.fields;
+
+        fields.forEach(function (el, i) {
+
+            let obj = {
+                ...el, key: i
+            }
+
+            switch (el.type) {
+                case "text":
+                    childNodes.push(<Input {...obj} />)
+                    break;
+                case "password":
+                    childNodes.push(<Password {...obj} />)
+                    break;
+                case "textarea":
+                    childNodes.push(<Textarea {...obj} />)
+                    break;
+                case "radio":
+                    childNodes.push(<Radio {...obj} />)
+                    break;
+                case "select":
+                    childNodes.push(<Select {...obj} />)
+                    break;
+                default:
+                    childNodes.push(<input {...obj} />)
```

Comment:
On line 26 you are using custom Input component, but here you have the default. I think you can use the same component. 
Also, please add Custom in front of all custom components to avoid confusion (Input -> CustomInput...)